packages/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.style.ts

```diff
@@ -30,14 +30,20 @@ export const htmlRenderLayerNavigationStyles = (context, definition) => css`
         margin: calc(${FocusOutlineWidthProperty} * -1px) 0 0
             calc(${FocusOutlineWidthProperty} * -1px);
     }
-    .click-layer.active {
+    .navigation-select__insetVert {
```

Comment:
I would just use the whole word, or change it to x/y:

```suggestion
    .navigation-select__insetY {
```

Reply:
Done.